docs/source/example_notebooks/gcm_linear_scm_bootstap_pvalues/false_discovery_rate_control.py

```diff
@@ -0,0 +1,152 @@
+"""Functions in this module should be considered experimental, meaning there might be breaking API changes in the
```

Comment:
Can you move the FDR implementation(s) into a separate own PR? By separating it, the PR is easier to check and the contribution is clearer.